### PR TITLE
CLN/COMPAT: for various py2/py3 in doc/bench scripts

### PR DESCRIPTION
**Pull request by cclauss**
- [ ] closes #xxxx
 - [ ] tests added / passed
 - [ ] passes ``git diff upstream/master -u -- "*.py" | flake8 --diff``
 - [ ] whatsnew entry

### CODE REVIEW

**Comment by jreback** [comment]
instead of this would be happy with progress on https://github.com/pandas-dev/pandas/issues/13704

blow away most of these scripts

**Comment by gfyoung** [comment]
> instead of this would be happy with progress on #13704

I agree with this.  Should we close then?

**Comment by cclauss** [comment]
#13704 has been open almost one year.  I can look into it but that is going to require me learning much more project context to make sure that useful code is not thrown away in the process.  The contents of this pull request are targeted changes to ensure that the entire project can pass a flake8 test.  I think that gives them value even if the files that they are in are deleted next week or next month.

**Comment by gfyoung** [comment]
> The contents of this pull request are targeted changes to ensure that the entire project can pass a flake8 test.

We actually don't check those files during `flake8`.

> I think that gives them value even if the files that they are in are deleted next week or next month.

That is fair.  I'm kind of indifferent given that there hasn't been much progress on this.  I'll leave it to @jreback to decide since he piloted the issue in the first place.

File: bench/bench_join_panel.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # reasonably efficient
 
+import numpy as np
+from pandas import Panel
```
Comment (jreback) [inline bench/bench_join_panel.py]: remove this entire directory! (bench), these are no longer used.

File: scripts/windows_builder/check_and_build.py [hunk 2]
```diff
@@ -49,7 +49,7 @@
 base_dir = "C:\Users\Jeff Reback\Documents\GitHub\pandas"
 remote_host='pandas.pydata.org'
 username='pandas'
```
Comment (jreback) [inline scripts/windows_builder/check_and_build.py]: remove the entire directory: scripts/windows_builder as these are not longer used

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by jreback** [comment]
@cclauss you don't need to make these separate commits. I am going to squash this anyhow.

**Comment by cclauss** [comment]
@jreback Yes, please squash!  Thanks.

**Comment by jreback** [comment]
So each time a push happens the CI's are all triggered. They do fail with duplicates (IOW a new push is spun up). You can certainly have many commits, just don't push after *each* one.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @cclauss

**Comment by jreback** [comment]
if you wanted to fix the remaining on this issue: https://github.com/pandas-dev/pandas/issues/13704

would be great.

**Pull request merged**